src/design-tokens/tier-1-definitions/colors.json

```diff
@@ -3,246 +3,130 @@
           "400": { "value": "#F7BE2D" },
-          "500": { "value": "#F79D2D" },
+          "500": { "value": "#D18400" },
           "600": { "value": "#BF7300" },
           "700": { "value": "#9E5B03" },
           "800": { "value": "#854C03" }
         },
         "lemon": {
-          "value": "#FCFF00"
+          "value": "#F5FF8F"
         },
         "orange": {
           "100": { "value": "#FFF1E9" },
-          "200": { "value": "#FFCBA6" },
-          "300": { "value": "#FFB077" },
+          "200": { "value": "#FFCBA5" },
+          "300": { "value": "#FFAF76" },
           "400": { "value": "#F6924A" },
-          "500": { "value": "#E06C00" },
+          "500": { "value": "#E06B00" },
           "600": { "value": "#C64600" },
           "700": { "value": "#AC3400" },
-          "800": { "value": "#852800" }
+          "800": { "value": "#842800" }
         },
         "ruby": {
-          "100": { "value": "#FFEBF0" },
-          "200": { "value": "#FFCCD8" },
+          "100": { "value": "#FFF0F4" },
+          "200": { "value": "#FFCBD7" },
           "300": { "value": "#FB90B0" },
           "400": { "value": "#F76C96" },
-          "500": { "value": "#F1497C" },
+          "500": { "value": "#F1497B" },
           "600": { "value": "#D41E52" },
           "700": { "value": "#BD0044" },
           "800": { "value": "#8F0134" }
         }
       },
-      "white": {
-        "value": "#FFFFFF",
-        "comment": "old token alias, remove with next major release"
-      },
-      "black": {
-        "value": "{eds.color.neutral.700.value}",
-        "comment": "old token alias, remove with next major release"
-      },
-      "alert": {
-        "100": {
-          "value": "{eds.color.other.ruby.100.value}",
-          "comment": "old token alias, remove with next major release"
-        },
-        "200": {
-          "value": "{eds.color.other.ruby.200.value}",
-          "comment": "old token alias, remove with next major release"
-        },
-        "300": {
-          "value": "{eds.color.other.ruby.300.value}",
-          "comment": "old token alias, remove with next major release"
-        },
-        "400": {
-          "value": "{eds.color.other.ruby.400.value}",
-          "comment": "old token alias, remove with next major release"
-        },
-        "500": {
-          "value": "{eds.color.other.ruby.500.value}",
-          "comment": "old token alias, remove with next major release"
-        },
-        "600": {
-          "value": "{eds.color.other.ruby.600.value}",
-          "comment": "old token alias, remove with next major release"
-        },
-        "700": {
-          "value": "{eds.color.other.ruby.700.value}",
-          "comment": "old token alias, remove with next major release"
-        },
-        "800": {
-          "value": "{eds.color.other.ruby.800.value}",
-          "comment": "old token alias, remove with next major release"
-        }
-      },
-      "success": {
+      "info": {
```

Comment:
keeping around the `"info"` and `"highlight"` colors for now, without changing the names, to reduce size of this migration